ToggleButton/ToggleButton.less

```diff
@@ -0,0 +1,84 @@
+// ToggleButton.less
+//
+@import "../styles/mixins.less";
+@import "../styles/fonts.less";
+@import "../styles/skin.less";
+
+.toggleButton {
+	.applySkins({
+		// .padding-start-end(@agate-button-h-padding, @agate-toggle-button-text-switch-gutter + @agate-button-h-padding + @agate-toggle-button-switch-width + @agate-toggle-button-switch-border-width);
```

Comment:
Can we remove this, if it's indeed cruft?

Reply:
yep